content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md

```diff
@@ -778,6 +778,7 @@ to hold the configuration of the NGINX server:
     $ kubectl delete deployment my-nginx -n mesh-external
     $ kubectl delete namespace mesh-external
     $ kubectl delete gateway istio-egressgateway
+    $ kubectl delete virtualservice nginx
```

Comment:
How did this change get cherrypicked here? We can cherrypick this too since this cleanup step was missing anyways.

Reply:
That is strange????